### PR TITLE
LITE-24436: pytest warnings fixed

### PR DESCRIPTION
**Pull request by gab832**
The idea is to remove all warnings that are displayed on the test result.

The usage of flake8 deprecated API was generating 672 Warnings.
tests/plugins/project/test_extension_helpers.py: 336 warnings
  /usr/local/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ast.py:407: DeprecationWarning: visit_Str is deprecated; add visit_Constant
    return visitor(node)
 
So I considered the following solutions:
1 - Find a replacement to keep the same features. I did not find anything.
2 - Hide the warnings because everything is working fine (chosen)
3 - Delete the code that is generating the warnings. It is testing that the generated files of the extension bootstrap is compliant with some flake8 standards. Because we are generating those files It might not be much useful. The responsibility to have certain flake8 standards will be for the extension developer.

Also we had a warning because we are using unsupported data validations on the openpyxl library, but everything is working well:
tests/plugins/product/test_clone.py: 4 warnings
tests/plugins/product/test_commands.py: 2 warnings
tests/plugins/product/sync/test_general.py: 14 warnings
tests/plugins/shared/test_base.py: 5 warnings
tests/plugins/shared/test_translation_sync.py: 18 warnings
tests/plugins/shared/test_utils.py: 1 warning
  /Users/galonso00/.env/connect-cli/lib/python3.9/site-packages/openpyxl/worksheet/_reader.py:312: UserWarning: Data Validation extension is not supported and will be removed
    warn(msg)

I decided the hide the warnings also.

### CODE REVIEW

**Comment by d3rky** [comment]
Hey, Gerard! Do you think it is a good idea to hide warnings in tests? I mean, if there is a deprecated warning, probably we can upgrade the version, like for flake8 plugin, or contribute to the plugin? I think hiding warnings should be a w/a solution with a clear path on how to resolve it in the future. WDYT?

**Comment by gab832** [comment]
About the flake8 warnings I think that it is not the best solution, but I did not find any alternative. From my point of view I would leave it hided and when they deprecate it, they might have an alternative solution. But take into account that we are testing that the code that we are generating in the extension bootstrap project complies with the flake8 checks. I mean, this should be the responsibility of the extension owner developer and maybe he has not flake8 checks. With all these, I would suggest to delete this small part of the code.

About the other warnings related to the data validation used in openpyxl, I did not also find any solution and the code works well, so I considered that there's no reason to show that warning.

In general terms I would be agree with you, hiding warnings could be dangerous, but if the warning is not helpful at all it does not make sense to show it. WDYT?

**Comment by d3rky** [comment]
OK, agree with you

File: connect/cli/core/utils.py [hunk 1]
```diff
@@ -3,10 +3,9 @@
 # This file is part of the Ingram Micro Cloud Blue Connect connect-cli.
 # Copyright (c) 2019-2022 Ingram Micro. All Rights Reserved.
 import os
-import re
 from collections import OrderedDict
-from distutils.version import StrictVersion
 
+from packaging.version import InvalidVersion, Version
```
Comment (d3rky) [inline connect/cli/core/utils.py:8]: Can you also explain what is the reason for these changes? Do they correlate with hiding the warnings?

**Review by d3rky — CHANGES_REQUESTED** [review]

File: connect/cli/core/utils.py (hunk 1 above)
Comment (gab832) [reply connect/cli/core/utils.py:8]: No, this was the 3d warning group. In this case, in the warning they suggest to use this build-in module instead of the deprecated distutils.

**Review by d3rky — APPROVED** [review]
